Grid/Grid.php

```diff
@@ -283,7 +283,7 @@ public function createConfigurationResponse(): JsonResponse {
 			$view->jsonConfiguration['columnDefs'][] = $column->jsonConfiguration;
 		}
 
-		$template = '{"gridId": ' . json_encode($view->getGridId()) . ', "aggrid": ' . json_encode($view->jsonConfiguration) . ', "stinger": ' . $jsonStinger . '}';
+		$template = '{"gridId": ' . json_encode($view->getGridId()) . ', "aggrid": ' . json_encode($view->jsonConfiguration) . ', "stinger": ' . $jsonStinger . ', "additionalAjaxRequestBody": '. json_encode($this->options['additionalAjaxRequestBody']) .'}';
```

Comment:
the name of the parameter may not be "additionalAjaxRequestBody" because it is open to the user. On top of that, will this throw an exception if the option isn't available?

Reply:
Reverting this change, because data should be persistent in JS, so the backend can use this data for further progress. Therefore I adapt the StingerConfigs. Hope this fits...